src/main/java/seedu/parser/Parser.java

```diff
@@ -228,8 +234,6 @@ public Item parseItemForAdding(String userInput) throws FridgetException {
             long daysPast = ChronoUnit.DAYS.between(expiryDate, LocalDate.now());
             errorMessage = "[" + itemName + "]" + " expired " + daysPast + " days ago.";
             throw new FridgetException(errorMessage);
-        } else if (ChronoUnit.CENTURIES.between(expiryDate,LocalDate.now()) > 1) {
-            throw new FridgetException(itemName + " expires more than a century later.");
```

Comment:
I think I'll create a warning but still allow the user to add the thing in my PR.